zulip_bots/setup.py

```diff
@@ -5,6 +5,10 @@
 
 import sys
 
+# Before anything, generate MANIFEST.in
+import generate_manifest
+generate_manifest.main()
```

Comment:
I don't think its a big deal, honestly! Ideally, we would do this as part of a custom build step in `setup.py`. :)